db/schema.rb

```diff
@@ -53,9 +63,14 @@
     t.datetime "updated_at", null: false
   end
 
+  add_foreign_key "comments", "posts"
   add_foreign_key "comments", "posts", column: "posts_id"
+  add_foreign_key "comments", "users", column: "author_id"
   add_foreign_key "comments", "users", column: "users_id"
+  add_foreign_key "likes", "posts"
   add_foreign_key "likes", "posts", column: "posts_id"
+  add_foreign_key "likes", "users", column: "author_id"
   add_foreign_key "likes", "users", column: "users_id"
+  add_foreign_key "posts", "users", column: "author_id"
   add_foreign_key "posts", "users", column: "users_id"
```

Comment:
- [ ] Here you have added column as `author_id` but you have not removed `user_id`. This is creating issue while creating posts, comments, or likes. Please fix this. First you have to rollback and then migrate the right command.